Dockerfile

```diff
@@ -1,42 +1,28 @@
-FROM alpine:3.8 AS stage1
+FROM ubuntu:jammy AS stage1
 
-# IPMITOOL
+# Dell racadm
+WORKDIR /tmp/racadm
 
-ARG IPMITOOL_REPO=https://github.com/ipmitool/ipmitool.git
-ARG IPMITOOL_COMMIT=19d78782d795d0cf4ceefe655f616210c9143e62
+## Install Dependencies
+RUN apt update && apt install -y libssl-dev wget pciutils libargtable2-0
 
-WORKDIR /tmp
-RUN apk add --update --upgrade --no-cache --virtual build-deps\
-            alpine-sdk \
-            automake \
-            autoconf \
-            libtool \
-            openssl-dev \
-            readline-dev \
-    && git clone -b master ${IPMITOOL_REPO}
+## Download
```

Comment:
Maybe just add a comment here on why we're setting the user-agent (Dell filters the default one from wget)